experimental/dds/tree2/src/test/feature-libraries/default-field-kinds/optionalField.spec.ts

```diff
@@ -53,15 +57,25 @@ const failCrossFieldManager: CrossFieldManager = {
 	set: () => assert.fail("Should modify CrossFieldManager"),
 };
 
-const deltaFromChild1 = (child: NodeChangeset): Delta.Modify => {
+const deltaFromChild1 = (
+	child: NodeChangeset,
+	{ marks, handler }: RepairDataBuilder,
+	idAllocator: IdAllocator,
+): Delta.Modify => {
 	assert.deepEqual(child, nodeChange1);
+
+	const moveId = brandOpaque<Delta.MoveId>(idAllocator());
+	marks.set(handler({ revision: change1Revision, localId: brand(0) }), [
+		{ type: Delta.MarkType.MoveIn, count: 1, moveId },
```

Comment:
Don't forget the isDelete/isRemoval flag here and for the MoveOut, and for the deltaFromChild2 implementation.